code/datums/bodypart_overlays/texture_bodypart_overlay.dm

```diff
@@ -19,5 +19,6 @@
 	return "[type]"
 
 /datum/bodypart_overlay/texture/spacey
+	blocks_emissive = EMISSIVE_BLOCK_NONE
```

Comment:
we dont actually draw an icon and just modify other overlays so this wouldn't have an effect. still it has good fibes
(dont ask why this is a bodypart overlay)

Reply:
makes bodypart 
### overlay
system, immediately uses it for non-overlays

Reply:
I was lazy dw it’s on my todo list